runtime/compiler/control/J9Options.cpp

```diff
@@ -2081,8 +2081,24 @@ bool J9::Options::preProcessJitServer(J9JavaVM *vm, J9JITConfig *jitConfig)
          int32_t xxUseJITServerArgIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, xxUseJITServerOption, 0);
          int32_t xxDisableUseJITServerArgIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, xxDisableUseJITServerOption, 0);
 
-         // Check if option is at all specified
-         if (xxUseJITServerArgIndex > xxDisableUseJITServerArgIndex)
+         bool explicitClientMode = xxUseJITServerArgIndex > xxDisableUseJITServerArgIndex;
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+         J9VMThread *currentThread = vm->internalVMFunctions->currentVMThread(vm);
```

Comment:
Suggest to share `vm->internalVMFunctions` via a local variable.

Reply:
Fixed, thanks.